gpswearos/src/main/java/com/android/gpswearos/presentation/MainActivity.kt

```diff
@@ -0,0 +1,104 @@
+/* While this template provides a good starting point for using Wear Compose, you can always
+ * take a look at https://github.com/android/wear-os-samples/tree/main/ComposeStarter and
+ * https://github.com/android/wear-os-samples/tree/main/ComposeAdvanced to find the most up to date
+ * changes to the libraries and their usages.
+ */
+
+package com.android.gpswearos.presentation
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Devices
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.wear.compose.material.Card
+import androidx.wear.compose.material.Text
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            StatusCard(SampleData.statusSample)
```

Comment:
We should use `Scaffold` here as the top level Composable for the basic infrastructure of the Wear app - see https://youtu.be/B7D3G6tC9n0?t=905. This also lets you include `timeText` at the top of the screen, and gives you a position indicator for scrolling a ScalingLazyColumn.

Reply:
A good code lab where you can see a complete Wear Compose app with Theme and Scaffold is https://developer.android.com/codelabs/compose-for-wear-os#0. A corresponding code-along video is https://www.youtube.com/watch?v=7U3FwzuqgRE.

Reply:
> We should use `Scaffold` here as the top level Composable for the basic infrastructure of the Wear app - see https://youtu.be/B7D3G6tC9n0?t=905. This also lets you include `timeText` at the top of the screen, and gives you a position indicator for scrolling a ScalingLazyColumn.
> 
> We can skip the whole navigation configuration for now because for the minimum viable product (MVP) of this Wear app it will just be a single screen that simulates what you see in the "Status" tab in the phone app.

Thanks, I've learned this and applied it.